Roche-Engine/TileMap/TileMapPaintOnMap.cpp

```diff
@@ -85,9 +107,9 @@ void TileMapPaintOnMap::HandleEvent(Event* event)
 		m_fCameraX = m_camera->GetPosition().x - m_camera->GetInitPosition().x + m_iStartingPosX;
 		m_fCameraY = m_camera->GetPosition().y - m_camera->GetInitPosition().y + m_iStartingPosY;
 
-		Vector2f mousePos = *static_cast<Vector2f*>(event->GetData());
-		m_iTileX = (mousePos.x + m_fCameraX) / 32;
-		m_iTileY = (mousePos.y + m_fCameraY) / 32;
+		m_fMousePos = *static_cast<Vector2f*>(event->GetData());
+		m_iTileX = (m_fMousePos.x + m_fCameraX) / 32;
+		m_iTileY = (m_fMousePos.y + m_fCameraY) / 32;
```

Comment:
Was just thinking that I can probably just update this to be used in the release/simulated release modes, but might need a video showing how to "plant" and spawn enemies so that I can test in those modes and ensure that it's working properly.

Reply:
Mouse Pos is not correct in release/simulated mode. Change the tile map layer to Foreground and then un-tick draw continuously and click near the player, after 30seconds the enemies will spawn. You can also select different seed packets (seed selection only worked with scroll wheel when testing) and it will change the tilemap texture to be that plant, and a different enemy will spawn 
![image](https://user-images.githubusercontent.com/57526297/216935717-39f04e07-b1bb-4b1f-8c1d-f7c911fdabea.png)